cloudprem/outputs.tf

```diff
@@ -5,10 +5,10 @@ output "eks_cluster_access_role" {
 
 output "dashboard_url" {
   description = "URL to your Dozuki Dashboard."
-  value       = "${module.nlb.this_lb_dns_name}:8800"
+  value       = format("https://%s:8800",module.nlb.this_lb_dns_name)
```

Comment:
Does `terraform fmt` not balk at the lack of whitespace here? Might be outdated....

Reply:
Lack of whitespace where exactly? After the parameter comma?

Reply:
Yep! `terraform fmt` doesn't complain about that?